manage/filters.js

```diff
@@ -293,7 +293,7 @@ function showFiltersStats({immediately} = {}) {
   }
   $('#filters').classList.toggle('active', filtersSelector.hide !== '');
   const numTotal = BG.cachedStyles.list.length;
-  const numHidden = installed.getElementsByClassName('entry hidden').length;
+  const numHidden = $$('.entry.hidden', installed).length;
```

Comment:
This code may be executed a lot while the user types something in the search input field. Even though this is a rare scenario overall, but it's better to keep getElementsByClassName as it's much-much-much faster than querySelectorAll used in $$.